src/gluonts/mx/block/quantile_output.py

```diff
@@ -20,13 +20,46 @@
 from gluonts.mx import Tensor
 
 
+def uniform_weights(x: list) -> List[float]:
+    return [1.0 / len(x) for _ in x]
+
+
+def crps_weights_pwl(quantile_levels: List[float]) -> List[float]:
+    """
+    Computes the quantile loss weights corresponding to CRPS under linear
+    interpolation assumption.
+
+    Quantile levels are assumed to be sorted in increasing order.
+
+    CRPS = sum_{i=0}^{n-1} 0.5 * (q_{i+1} - q_{i}) * (z_{i+1} + z_{i})
+    under the assumption of linear interpolation, where z_i is the
+    ith quantile prediction q_i. The inner terms cancel due to the
+    telescoping sum property and we obtain CRPS = sum_{i=1}^n w_i z_i, with
+    the weights w_i = (q_{i+1}-q_{i-1})/2 for i = 1, ..., n-1,
+    w_0 = (q_1-q_0)/2 and w_n = (w_n - w_{n-1})/2.
+    """
+    num_quantiles = len(quantile_levels)
+    assert num_quantiles >= 0
```

Comment:
```suggestion
    assert quantile_levels
```

Reply:
Same here

Reply:
Actually, this check doesn't make sense at all. Are we ensuring that `len` decides to return negative values?

Reply:
One is never too sure! (Joking, I'll remove it)